### PR TITLE
Add RBAC Action listbackups API for sites

### PR DESCRIPTION
**Pull request by naveedaz**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [X] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json [hunk 1]
```diff
@@ -12103,6 +12148,58 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/listbackups": {
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Gets existing backups of an app.",
```
Comment (yungezz) [inline specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json]: duplicate summary and description with above  operation

File: specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json [hunk 2]
```diff
@@ -4585,6 +4585,51 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/listbackups": {
```
Comment (yungezz) [inline specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json]: would you like to add examples for those operations?

File: specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json [hunk 3]
```diff
@@ -4585,6 +4585,51 @@
...
+      "post": {
```
Comment (yungezz) [inline specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json]: adding new apis in existing stable versions is breaking change, suggest to add in new release

Comment (naveedaz) [reply specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json]: Why is adding a new API a breaking change? Existing APIs are untouched.

Comment (KrisBash) [reply specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json]: Not breaking

File: specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json [hunk 4]
```diff
@@ -12103,6 +12148,58 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/listbackups": {
```
Comment (KrisBash) [inline specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json]: The APIs look fine, but what is the need to expose these as POST actions? I assume they are not privileged and there are equivalent GET apis already. Is there really a need for the alternate method?

**Review by KrisBash — CHANGES_REQUESTED** [review]
APIs are modeled fine, but please see question about need

File: specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json (hunk 4 above)
Comment (naveedaz) [reply specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json]: GETs right now expose secrets, so we need the post for RBAC.

**Review by KrisBash — APPROVED** [review]
LGTM

**Review by raych1 — CHANGES_REQUESTED** [review]
@naveedaz , can you fix the comment of duplicated `summary` and `description` for these four operations?
https://github.com/Azure/azure-rest-api-specs/blob/6cae6338e6c3670fea0f5c3db7deb9f677d86929/specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json#L442
https://github.com/Azure/azure-rest-api-specs/blob/6cae6338e6c3670fea0f5c3db7deb9f677d86929/specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json#L7501
https://github.com/Azure/azure-rest-api-specs/blob/6cae6338e6c3670fea0f5c3db7deb9f677d86929/specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json#L4593
https://github.com/Azure/azure-rest-api-specs/blob/6cae6338e6c3670fea0f5c3db7deb9f677d86929/specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json#L12156

**Comment by naveedaz** [comment]
Added the new API in a newer API version.

**Pull request closed without merging**